packages/swap/src/utils/broker.ts

```diff
@@ -26,8 +48,20 @@ const requestValidators = {
       chainflipAsset.transform(transformAsset),
       z.union([numericString, hexString, btcAddress]),
       z.number(),
+      ccmMetadataSchema
+        .merge(
+          z.object({
+            source_chain: chainflipChain,
+            source_address: z.literal(0),
+          }),
+        )
+        .optional(),
     ])
-    .transform(([a, b, c, d]) => [a, b, c, d]),
+    .transform(([a, b, c, d, e]) =>
+      [a, b, c, d, transformObjToSnakeCase(e)].filter(
+        (item) => item !== undefined && item !== null,
```

Comment:
Let's add this to the `shared/guards.ts`:
```ts
export const isNotNullish = <T>(value: T | null | undefined): value is T =>
  value !== null && value !== undefined;
```

```suggestion
        isNotNullish,
```